tests/wpt/meta/html/dom/reflection-forms.html.ini

```diff
@@ -3551,18 +3533,6 @@
   [progress.max: setAttribute() to "+100"]
     expected: FAIL
 
-  [progress.max: IDL set to -10000000000]
-    expected: FAIL
-
-  [progress.max: IDL set to -1]
-    expected: FAIL
-
-  [progress.max: IDL set to -0]
-    expected: FAIL
-
-  [progress.max: IDL set to 0]
-    expected: FAIL
-
   [progress.max: IDL set to 1e-10]
```

Comment:
I think there are even more tests that we could fix with this (look for `IDL set to x`).

Reply:
Thank you - I did have a longer look into those. I think there's an edge case in our implementation of `is_valid_floating_point_number_string` that fails for `1.e2`. I'm pretty sure this one should be valid, but our regex enforces that a digit appears after the decimal point.

https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#valid-floating-point-number